### DISCUSSION TITLE
Jamulus server installation via ansible

### DISCUSSION THREAD

**Discussion by moan0s** [jamulussoftware/jamulus#2942] [General]
Together with a friend I pieced together an ansible install script for jamulus. It also features some documentation on how to setup your server, domain etc...

I don't know if this will be useful to anyone, for me it is. Feel free to open issues when you see room for improvement :)
https://github.com/moan0s/ansible-jamulus

**Comment by ann0see** [comment]
Hi,

Sorry for the late reply. Although I don’t use Ansible, I‘ve heard that it‘s very useful for sys admins, so big thank you. 

Feel free to add your script to the related projects page: https://jamulus.io/kb/2022/01/25/Related-Projects.html in the Website Repo: https://github.com/jamulussoftware/jamuluswebsite/edit/release/_posts/2022-01-25-Related-Projects.md
